benchmark/benchmark/base.py

```diff
@@ -154,13 +154,13 @@ def args(self) -> argparse.Namespace:
     def class_params(self) -> Dict[str, Any]:
         return self._class_params
 
-    def train_df(
-        self, spark: SparkSession
+    def input_dataframe(
+        self, spark: SparkSession, *paths: str
```

Comment:
Should probably rename the `train_df` variable on L197-L198 as well.

Reply:
Done. Thx